cmake/config/Config.cmake.in

```diff
@@ -145,7 +145,7 @@ set(DEAL_II_MPIEXEC_POSTFLAGS "@MPIEXEC_POSTFLAGS@")
 # Information about git revision, library targets and feature configuration
 #
 
-set(DEAL_II_GIT_CONFIG "${DEAL_II_PATH}/${DEAL_II_PROJECT_CONFIG_RELDIR}/${DEAL_II_PROJECT_CONFIG_NAME}Git.cmake")
+set(DEAL_II_GIT_CONFIG "${DEAL_II_PATH}/${DEAL_II_PROJECT_CONFIG_RELDIR}/${DEAL_II_PROJECT_CONFIG_NAME}ConfigGit.cmake")
```

Comment:
How did this work before?

Reply:
I would guess that this file is just not regularly used.

Reply:
@bangerth This did not work before. But this is how it is intended to be used (sans workaround): https://github.com/conservation-laws/ryujin/commit/afdc795dce33d0558d5561128e5074480d703407